### DISCUSSION TITLE
How to change commit author?

### DISCUSSION THREAD

**Discussion by eduludi** [WeblateOrg/weblate#7341] [Q&A] [answered]
In the [documentation (under FAQ)](https://docs.weblate.org/no/latest/faq.html#why-are-all-commits-committed-by-weblate-noreply-weblate-org) says it's possible to change the default committer name (noreply@weblate.org) somewhere in the administration _"at any time"_:

<img width="742" alt="change commit author docs" src="https://user-images.githubusercontent.com/76132/156539630-746dee96-f799-4417-81d0-cd1f6166f87f.png">

I, logged in as `admin`, wasn't able to find where: its not in the component settings, nor in the project, not the in the admin area.

Any ideas? Can someone tell me where is this? Or how can I configure it? 

I'm running Weblate's latest version (`4.11`), in a Docker installation.

Also, I did configure `WEBLATE_DEFAULT_COMMITER_EMAIL` in `docker-compose-https.override.yml` like is documented [here](https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_DEFAULT_COMMITER_EMAIL), but it keeps using `noreply@weblate.org` as commits' author.

Thanks!

**Comment by nijel** [comment, marked as answer]
The documentation is wrong, as this is no longer present in the component configuration. I've fixed the docs in 3c784ca12e.

There are two things in git - author and committer. The committer is configured by settings. The author is the actual author of the change in Weblate unless the change has no actual author (anonymous suggestions or machine translation output). I've clarified this in cc29fa3ed5178628946489829763d7be709dd570.

**Reply by eduludi** [reply]
Thanks for your answer! 

Yes, I was aware of this, but we are getting the opposite: `Weblate <noreply@weblate.org>` as author and the translator is the commiter.

<img width="433" alt="Author and Commiter" src="https://user-images.githubusercontent.com/76132/156648013-d1c1c7a6-d90c-4687-9a3d-ee55381aa588.png">

With your answer I realised that we have the ["Add missing languages" Add-on ](https://docs.weblate.org/en/weblate-4.11/admin/addons.html#add-missing-languages) and in that case Weblate is the author of the commit as it's the one authoring the new language files.

Hope this helps someone else having this issue.

**Reply by nijel** [reply]
That's one of the changes which can not be credited to the real user, so it's expected. The committer might come from rebasing when merging these changes...
